src/main/kotlin/com/terraformation/backend/accelerator/db/ParticipantProjectSpeciesStore.kt

```diff
@@ -46,6 +48,36 @@ class ParticipantProjectSpeciesStore(
     return row.toModel()
   }
 
+  fun createMany(projectIds: Set<ProjectId>, speciesIds: Set<SpeciesId>): Unit {
```

Comment:
We can combine this and `updateMany` in the later stack to be `upsertMany` if do on conflict update status. But either way is fine!

Reply:
My thought was similar, that this could be combined with `deleteMany` (the `update` function sets other fields like feedback, so it needs to be separate). But it really depends on what the usage pattern is going to look like. If the UI has explicit "add" and "delete" operations, rather than "add/delete things and then submit all the changes at once," then this split makes sense.

Reply:
The UI has:
- assign X species to Y projects
- update single species with rationale, feedback, or status
- delete species(single and bulk) from project

"update all species statuses" was removed

I'm not sure I follow how to combine this with `deleteMany`. They seem like to distinct use cases.

Reply:
If the UI around this allowed adding and removing species at the same time like we do in other places where we edit lists of things, then doing it as a single "make the list of values look like this, inserting new ones and deleting existing ones that are no longer wanted" operation would make sense. But this way makes more sense for explicit add and delete operations.

Reply:
I see, I suppose it could go either way. I like this way a little better because it allows the UI to only be concerned with IDs, as opposed to managing a list of things. I will keep it like this for now and if we do need more robust list manipulation I can combine them.